package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "eslint src/*.js i18n/*.js",
     "replace": "node ./scripts/replace.js",
-    "compile": "browserify -d src/index.js -s osrm > bundle.raw.js && uglifyjs bundle.raw.js -c -m --source-map=bundle.js.map -o bundle.js",
+    "compile": "browserify -d src/index.js -s osrm > bundle.raw.js && uglifyjs bundle.raw.js -c -m --source-map filename=bundle.js.map -o bundle.js",
```

Comment:
I'm not sure `--source-map filename=bundle.js.map` option should contain whitespace, maybe dash instead?

Reply:
Well see the full comment on commit https://github.com/Project-OSRM/osrm-frontend/pull/370/commits/6d175909a5f1f7dfb01851223057ab5680042f4b

Apparently the command line behaviour changed. --source-map now takes a string of key value pairs. I found https://stackoverflow.com/questions/15590702/how-to-get-minified-output-with-browserify where someone uses three key-value pairs for that argument.